src/main/scala/com/cloudera/sparkts/ARIMA.scala

```diff
@@ -607,7 +602,6 @@ class ARIMAModel(
       // copy into results
       results(-(d + nFuture) to -1) := forwardIntegrated
     }
-    results
```

Comment:
Nah I'm wrong here. Yes really all that's needed is to not say `results` in the `if` block since it will be a no-op.